### ISSUE TITLE
docker command has issues with accented characters in pathnames

### ISSUE THREAD

**Issue by DogPawHat** [boot2docker/boot2docker#911]
So my Windows name is the same as my first name - Ciarán - with the accented a "á". When installing boot2docker on my system, I got 
```
Writing C:\Users\Ciarán\.boot2docker\certs\boot2docker-vm\ca.pem
Writing C:\Users\Ciarán\.boot2docker\certs\boot2docker-vm\cert.pem
Writing C:\Users\Ciarán\.boot2docker\certs\boot2docker-vm\key.pem
```
out of Powershell So far so good, but when I run ``docker run hello world`` I get 
```
FATA[0000] Couldn't read ca cert C:\Users\Ciar├ín\.boot2docker\certs\boot2docker-vm\ca.pem: open C:\Users\Ciar├ín\.boot2docker\certs\boot2docker-vm\ca.pem: The system cannot find the path specified.
```
With the á mangled. Can you give me feedback as to how to resolve this issue

**Comment by cgocmenoglu** [comment]
I have the same problem. I have Turkish characters in my user folder name (such as "ğ") and can't start boot2docker on Windows 8.

**Comment by DogPawHat** [comment]
I mangaed to find a workaround by using Git Bash on windows instead Powershell:
```
$ boot2docker up
$ eval $(boot2docker shellinit)
```

 Apparently PS translateds parameters to ascii before piping them into .exe

http://blogs.msdn.com/b/powershell/archive/2006/12/11/outputencoding-to-the-rescue.aspx

**Comment by thaJeztah** [comment]
@ahmetalpbalkan ^^ is that a known issue on your side?

**Comment by ahmetalpbalkan** [comment]
Hmm could be something wrong at quite many points (b2d cli, virtualbox, msys). 

@dogpawhat Can you  please run "boot2docker shellinit" in Git Bash and screenshot the output? Also can you please "boot2docker ssh" and see what the folder name appears like in /Users (also screenshot please).

**Comment by DogPawHat** [comment]
![capture](https://cloud.githubusercontent.com/assets/2125961/8026261/8a08cfe2-0d67-11e5-9a60-ae921bdef589.PNG)
Here this should have everything you need

**Comment by DogPawHat** [comment]
This is what my users folder looks like at the moment

![users](https://cloud.githubusercontent.com/assets/2125961/8026302/84449630-0d68-11e5-9a13-be01026d06c5.PNG)

Same folder in both Ciarán and Ciar?n
![ciaranvmfolder](https://cloud.githubusercontent.com/assets/2125961/8026304/88bfc19e-0d68-11e5-941f-39d614dbc53f.PNG)
![capture12](https://cloud.githubusercontent.com/assets/2125961/8026305/8b77636a-0d68-11e5-9640-411153e63d5b.PNG)

**Comment by ahmetalpbalkan** [comment]
Yeah apparently  a shared folders issue. I'm not familiar with the topic, let's start another issue at boot2docker/boot2docker repo. 

This does not look like a b2d cli problem.

**Comment by DogPawHat** [comment]
I deleted Ciar?n entirely, ran boot2docker delete and boot2docker init again, Ciar?n popped up again with the same spilt.
